lib/norm/core/any_of.ex

```diff
@@ -1,4 +1,4 @@
-defmodule Norm.Spec.Union do
+defmodule Norm.Core.AnyOf do
```

Comment:
what's the difference between AnyOf and Alt?

Reply:
`Alt` requires that all specs are tagged with a key.